index.html

```diff
@@ -0,0 +1,194 @@
+          <li class="login">
+            <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="paragraphe">
+        <div class="paragraphe_title">
+          <h3>Welcome to your best car's sale company</h3>
+          <h1>Hyundai Rwanda. Sales and Marketting Place</h1>
+
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo
+            doloremque tempora voluptas maxime? Nulla ex, magni odit nihil
+            aspernatur itaque fugiat numquam odio veritatis placeat perferendis,
+            enim suscipit quas ipsa blanditiis! Et consequatur reiciendis
+            temporibus, animi earum ipsa id distinctio dolor tempora mollitia,
+            similique?
+          </p>
+          <div class="span_para">
+            <h4>07 FEB 2022(Monday) - 16(Wednesday) CAR SOLD</h4>
+          </div>
+          <small
+            >@National center of car production, sales and transportation</small
+          >
+        </div>
+      </section>
+
+      <section id="courses" class="my-team">
+        <p>Executive Team</p>
+        <hr />
+        <ul class="team">
+          <li class="team-list">
+            <img src="./images/user1.jpg" alt="" class="flex1" />
+            <p class="title">CEO</p>
+            <p class="flex3">
+              Executive founder of Hyundai based in North-Korea
+            </p>
+          </li>
+          <li>
+            <img src="./images//seccretaire.jpg" alt="" class="flex1" />
+            <p class="title">Advisor</p>
+            <p class="flex3">
+              Magistral Advisor of the company <br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img src="./images/user2.jpg" alt="" class="flex1" />
+            <p class="title">COO</p>
+            <p class="flex3">
+              Co-founder of Hyundai Brand based in Ethiopia<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img
+              src="./images/pexels-orione-conceição-7527617.jpg"
+              alt=""
+              class="flex1"
+            />
+            <p class="title">Marketing</p>
+            <p class="flex3">
+              Agen marketing specialist with PHD<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img src="./images/msrket.jpg" alt="" class="flex1" />
+            <p class="title">Secretary</p>
+            <p class="flex3">
+              Executive secretary of the Brand<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+        </ul>
+        <button type="button" class="orderNow">ORDER YOUR CAR NOW</button>
+        <button type="button" class="open-details">SEE THE WHOLE SALES</button>
+      </section>
+
+      <section class="cars"></section>
+    </main>
+
```

Comment:
- [ ] It would be nice to follow the design layout for this section. The images for this section are placed on a **_chessboard_** like image. Kindly implement it as it is part of the design that makes the application dynamic.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/153725241-7488cd39-821f-43f4-b5e2-d7219ef60e23.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/153725267-38310ac6-62ec-48c3-bba8-c502c0afdd51.png)